### ISSUE TITLE
Add support for scrolling things other than `window`

### ISSUE THREAD

**Issue by wesleytodd** [StreamMeDev/scroll-to#6]
Something like this:

``` javascript
scrollTo({
  element: document.querySelector('.some-element'),
  x: 100
});

```

**Comment by cwebley** [comment]
Was just thinking about this too. Not extremely familiar with this, but it looks like a slightly different api:

window
```javascript
// scroll to top left of page
window.scroll(0, 0)
```

element
```javascript
// set element scroll position to top left:
ele.scrollLeft = 0;
ele.scrollTop = 0;
```

**Comment by wesleytodd** [comment]
hmm, yeah seems like more work than I thought.  I still think that the exposed api could be like I said above, but the implementation itself might have to change.  I was originally thinking it would just be:

```javascript
options.element = options.element || window;
// ...
options.element.scroll();
```

**Comment by cwebley** [comment]
Yeah that's what I was hoping too. Anyway, I can do this after I finish up some other work this week if you think we'll use element scrolling soon

**Comment by wesleytodd** [comment]
No rush, but it would be nice to use this in the chat's "scroll to bottom" button

**Comment by wesleytodd** [comment]
Closed in #7
